test/ios/getProjectConfig.spec.js

```diff
@@ -1,31 +1,25 @@
-const path = require('path');
-const expect = require('chai').expect;
+jest.autoMockOff();
+
 const getProjectConfig = require('../../src/config/ios').projectConfig;
 const mockFs = require('mock-fs');
 const projects = require('../fixtures/projects');
 
 describe('ios::getProjectConfig', () => {
+  const userConfig = {};
 
-  before(() => {
-    mockFs({ testDir: projects });
-  });
+  beforeEach(() => mockFs({ testDir: projects }));
```

Comment:
to be honest I haven't revisited test's logic, just replaced some technical aspects. But I can take a look